comics/comics/anleggsplassen.py

```diff
@@ -30,7 +29,7 @@ def crawl(self, pub_date):
             date_string = article_page.content(
                 'meta[property="article:published_time"]'
             )
-            date = dateutil.parser.parse(date_string).date()
+            date = datetime.strptime(date_string[:10], "%Y-%m-%d").date()
```

Comment:
You can use `date = self.string_to_date(date_string[:10], "%Y-%m-%d")` and avoid the import.